word_count.rb

```diff
@@ -0,0 +1,21 @@
+#Generates the count of each word in the file sorted by the maximum occured word first
+def wordCount
+  file = File.open("speech.txt")
+
+  words = file.read().downcase().split(/\W+/)
+  words_map = {}
```

Comment:
Anytime you instantiate an empty collection in order to iteratively build it up is a good clue that you should probably look to use a method like `.map` or `.reduce`